mkosi/__init__.py

```diff
@@ -3261,7 +3261,99 @@ def make_disk(
 
 
 def make_oci(context: Context, root_layer: Path, dst: Path) -> None:
-    raise NotImplementedError
+    ca_store = dst / "blobs" / "sha256"
+    ca_store.mkdir(parents=True)
```

Comment:
Use `with umask` here to make sure the mode is correct. Should be good with 755 here.

Reply:
Do you mean `os.umask` or the mode parameter of mkdir?